content/en/service_management/workflows/private_actions/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Private Actions
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/connections"
+  tag: "Documentation"
+  text: "App Builder Connections"
+- link: "service_management/workflows/connections"
+  tag: "Documentation"
+  text: "Workflow Connections"
+- link: "service_management/workflows/private_actions/private_action_credentials"
+  tag: "Documentation"
+  text: "Handling Private Action Credentials for Workflow Automation"
+- link: "service_management/app_builder/private_actions/private_action_credentials"
+  tag: "Documentation"
+  text: "Handling Private Action Credentials for App Builder"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSc_AEtxd8MPHUFyYIbX3hWEVF18iSWuk7kmA0PtlOPgo0pi3w/viewform" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Private actions allow your Datadog workflows and apps to interact with services hosted on your private network without exposing your services to the public internet. To use private actions, you must use Docker to install a private action runner on a host in your network and pair the runner with a Datadog Connection.
+
+The following integrations support private actions:
+- [Kubernetes][1]
+- [Postgres][2]
+- [Jenkins][3]
+- [Temporal][4]
+- The [HTTP][5] action
+
+## Prerequisites
+
+The private action runner requires a Linux host that is able to reach any internal services you want to call from an action or app.
+
+To use App Builder with private actions, you must be able to point a hostname to the private action runner container. This hostname must be resolvable by your App users. App Builder calls the hostname using HTTPS, so you must bring your own SSL termination.
```

Comment:
Is "the private action runner container" equivalent to "the container where the private action runner is installed" or "located" or something?